lib/lightstep/tracer.rb

```diff
@@ -28,7 +28,7 @@ class ConfigurationError < LightStep::Tracer::Error; end
     # @param access_token [String] The project access token when pushing to LightStep
     # @param transport [LightStep::Transport] How the data should be transported
     # @param tags [Hash] Tracer-level tags
-    # @param propagator [Propagator] :lightstep is the only supported option
+    #v@param propagator [Propagator] Symbol one of :lightstep, :b3 indicating the propgator
```

Comment:
Additional _v_ before `@param`

Reply:
Oh hey, there are two typos on this line. There's also a missing "a" in propagator.